### DISCUSSION TITLE
How to send or execute a command as player in client mod

### DISCUSSION THREAD

**Discussion by Panda98277** [FabricMC/fabric-api#3442] [Mod Dev Support]
### Description

I tried to use 'ClientPlayerEntity.networkHandler.sendChatMessage()'
but it can only send messages

**Comment by kevinthegreat1** [comment]
sendCommandMessage

**Reply by Panda98277** [reply]
I can't find this method
It seems to have been removed

**Comment by Panda98277** [comment]
In addition
version：mc1.20.2-build.1
